src/changeKeys.js

```diff
@@ -4,18 +4,29 @@ import pressedAnotherOperator from "./pressedAnotherOperator";
 const changeKeys = (result, button) => {
   const lastCharIsNum = "0123456789".includes(result[result.length - 1]);
   const lastCharIsOperator = "/*+-".includes(result[result.length - 1]);
+  const isOperator = "/*+-".includes(button);
 
   if (result === "") {
-    if ("+-(0123456789".includes(button)) {
+    if ("(0123456789".includes(button)) {
       return button;
     }
+    if (isOperator) {
+      return (result = "0" + button);
+    }
   }
   if (lastCharIsNum) {
     return ([result, button] = lastCharNumber(result, button));
   }
 
   if (lastCharIsOperator) {
-    return ([result, button] = pressedAnotherOperator(result, button));
+    if (".".includes(button)) {
```

Comment:
If you look at the commit with message "initialize result to zero for operations that begin with an operator", hopefully you'll agree there is a lot more going on in that commit that is unrelated to the commit message.

Reply:
Yes agreed!